src/collectors/rabbitmq/rabbitmq.py

```diff
@@ -238,13 +238,21 @@ def _publish_metrics(self, name, prev_keys, key, data):
         if isinstance(value, dict):
             for new_key in value:
                 self._publish_metrics(name, keys, new_key, value)
-        elif isinstance(value, (float, int, long)):
+        elif isinstance(value, (float, int, long, type(None))):
```

Comment:
What if we just did `value = data.get(key, 0)` on line 236? Is there a problem where we'd be reporting too many empty keys?